blog/announcing-new-branding/README.md

```diff
@@ -0,0 +1,97 @@
+# Post
+
+#### Title
+
+Announcing New Brand Identity
+
+#### Purpose
+
+Let our community know about our new brand identity.
+
+#### url
+
+blog.joystream.org/announcing-new-branding/
+
+#### Cover
+
+<p align="center"><img src="announcing-new-branding.png"></p>
+
+#### Lead
+
+We're very excited to announce the rollout of our new brand identity which will be taking place over the next couple of days!
+
+#### Body
+
+The Problem
+-----------
+
+Before the process of developing our new brand identity had begun, we were frustrated by the fact that the Joystream visual identity was lacking in many respects. It was incomplete and undefined, which led to problems in communication with our audience across all of our channels, products and messages.
+
+![](branding-collage.png)
+
+The problems we faced in this area included but were not limited to:
+
+-   The logo was inconsistent (sometimes it was our old elephant logo, sometimes a wordmark, sometimes an emblem and occasionally combinations of all three)
+-   Logo usage guidelines were unclear in terms of positioning, backgrounds, etc.
+-   Key brand colors and typefaces were undefined
+-   There were no rules for photo style or illustration style in our communications (e.g. blog posts)
+-   Icons and other assets lacked a consistent style
+-   Assets for certain purposes simply did not exist
+-   We had no established brand design guidelines or a style guide
+
+You can see [more of our discussion](https://github.com/joystream/design/issues?q=is%3Aissue+is%3Aclosed) about these issues in our design repo on GitHub.
```

Comment:
Consider [including open issues](https://github.com/joystream/design/issues?q=is%3Aissue) as well? 

Alternatively, you can add labels to the relevant ones (we should focus more on this anyway), and link to those.

Reply:
Will do!